README.md

```diff
@@ -1,7 +1,8 @@
 # LogDNA Agent
 
 LogDNA Agent streams from log files to your LogDNA account. Works with Linux, Windows, and macOS Servers.
-
+## Deprecation Warning!!!
```

Comment:
More than one exclamaition point is bad punctuation. it should really only ever be one.
maybe using `:warning:`


## :warning: DEPRECATION WARNING